### ISSUE TITLE
Butlin PP-1: qualify precision-weighted predictive coding with held-out adaptation and reversal

### ISSUE THREAD

**Issue by Tristan-Stoltz-ERC** [Luminous-Dynamics/symthaea#2066]
## Purpose

Define the indicator-specific PP-1 experiment after the shared predictive-processing prerequisites are repaired and qualified.

Dependencies:

- #2053 — reachable graded BinaryHV belief revision (draft; exact execution still required);
- #2056 — contextual precision must causally enter inference while preserving local layer precision;
- #2059 — perceptual prediction error must drive generative/recognition model plasticity rather than only transient belief revision.

These dependencies are **shared substrate evidence**, not PP-1 support by themselves.

## Target construct

PP-1 should establish a complete predictive-coding chain:

```text
prediction
    ↓
prediction error
    × precision
    ↓
latent/model update
    ↓
changed future prediction
    ↓
lower held-out prediction error when the environment is learnable
```

A nonzero learning rate, prediction-error scalar, or decreasing training-stream free energy is insufficient.

## Experimental world

Use a deterministic externally generated sequence world with at least three regimes:

1. **stable predictable** — a learnable cause/context → outcome relation;
2. **volatile/reversal** — the relation changes at a hidden boundary;
3. **irreducibly noisy** — no stable relation exists beyond declared chance/statistical structure.

Include held-out contexts/outcomes never used to choose thresholds or tune the learner.

The environment owns the sequence and ground truth. Symthaea must not receive a regime label or future change-point signal.

## Required arms

- adaptive production predictive stack;
- **error-frozen** intervention: expose the same observations but suppress the model-learning consequence of prediction error;
- **precision-frozen** intervention: preserve prediction error/model machinery but prevent adaptive precision changes;
- zero-effective-precision control for the relevant channel;
- matched unrelated sham;
- shuffled/broken cause→outcome pairing negative control;
- selective rescue restoring the missing error/precision/model-update signal without directly writing the correct prediction;
- optional fixed-statistical predictor baseline to ensure trivial frequency learning is not mistaken for hierarchical predictive coding.

## Primary outcomes

Pre-register at least:

- one-step held-out prediction error / similarity;
- calibration of prediction precision against actual correctness;
- model/association state change independent from latent belief state change;
- adaptation lag after hidden reversal;
- retention/interference on an unrelated learned relation;
- error-weighted update magnitude;
- cumulative regret or excess prediction error relative to an oracle appropriate to the synthetic world.

Do not use an opaque blended consciousness score.

### Stable regime

Adaptive PP should learn the relation and outperform frozen-error/model controls on held-out future prediction.

### Hidden reversal

Prediction error should rise at the change, then drive model revision; future error should recover. A permanently frozen model should remain wrong after reversal.

### Noisy regime

The system should not manufacture strong deterministic confidence or memorize noise indefinitely. Precision/calibration should reflect irreducible uncertainty.

### Precision manipulation

With the same prediction error, a verified change in effective precision must produce the documented change in update magnitude. Zero effective precision must block the corresponding update.

### Sham

An unrelated manipulation should not reproduce the targeted PP degradation pattern.

## Independence / anti-circularity

- Outcome ground truth comes from the external sequence generator, not Symthaea's own representation.
- Held-out prediction score must not be the same field used to verify intervention success.
- Model-change telemetry must be independent from the future-prediction outcome.
- The evaluator and expected directions are frozen before target results are inspected.
- A shared #2053/#2056/#2059 prerequisite is counted once in the evidence dependency graph.

## Replication ladder

Require, in order:

1. deterministic replay of exact seed/protocol;
2. fresh seeds;
3. fresh relation vocabulary / HDC identities;
4. structurally different predictive task with the same PP-1 construct;
5. alternate intervention implementation;
6. independent/external reproduction package before strong public claims.

## Evidence outcome rules

- qualified probe + predicted selective effect + validated intervention/sham/rescue → eligible causal evidence;
- qualified probe + no targeted effect → `NotDemonstrated`;
- wrong-direction robust effect → `Contradicted` where appropriate;
- failed manipulation / broken positive control / invalid task → `Inconclusive`;
- no single run should mint `FunctionallySupported` without the declared downstream/held-out consequence and replication level.

## Exit condition

A provenance-bound, preregistered full-system experiment demonstrates precision-weighted prediction error causing model adaptation that improves held-out future prediction, adapts after hidden reversal, remains calibrated under irreducible noise, survives shams, fails under targeted interventions, and is selectively rescued.

This is evidence for the **PP-1 indicator property**, not a binary consciousness claim.
